datastore/tests/integration/smoke_test.py

```diff
@@ -137,3 +141,41 @@
 
         after = await ds.runQuery(query, session=s)
         assert len(after.entity_results) == num_results + 3
+
+
+@pytest.mark.skipif('DATASTORE_EXPORT_BUCKET_NAME' not in os.environ,
+                    reason='DATASTORE_EXPORT_BUCKET_NAME not set')
+@pytest.mark.asyncio  # type: ignore
+async def test_datastore_export(creds: str, project: str,
+                                export_bucket_name: str):
+    kind = 'DatastoreExportTestModel'
```

Comment:
Use the `kind` fixture, please, or create a new fixture, but for now we want everything linked to the "public test" stuff.

Reply:
okay, i though it would be better to use a new kind just for this test. Why use the same kind for all?